tests/patch.go

```diff
@@ -0,0 +1,39 @@
+// From https://gist.github.com/imosquera/6716490#sthash.O4z2aQQp.LUHz2Cbb.dpuf
```

Comment:
@lpabon : Actually for vendoring we are using glide and it has a known flaw of pulling the complete parent dependencies. For that we chose to reduce the dependency as much as possible. If glide guys fix it in sometime which they promised we'll definitely start using these utilities from the origin instead of forking. Does it make sense?